app.js

```diff
@@ -18,11 +18,34 @@ app.use(session({
   saveUninitialized: true
 }));
 
+app.use('*', async (req, res, next) => {
+  if (req.session.user) {
+    res.locals.user = true;
+    
+    let user = await users.getUser(req.session.user.username);
```

Comment:
Do we want to catch any exceptions from this

Reply:
yeah, I can fix that